docs/connector.md

```diff
@@ -0,0 +1,51 @@
+# Connector
+
+The [`backend`](./architecture.md#backend) includes an [Interledger](./glossary.md#interledger-protocol) connector for sending and receiving [STREAM](./glossary.md#stream) packets.
```

Comment:
In there: https://github.com/interledger/rafiki/blob/65e316b6f786b9d0acd041cd0778944cf6a18613/docs/architecture.md?plain=1#L24 we should link this connector document